RELEASENOTES.md

```diff
@@ -1,5 +1,16 @@
 # Azure Monitor Metrics for AKS clusters
 
+## Release 01-31-2023
+
+* Linux image - `tbd`
+* Windows image - `tbd`
+* Change log -
+  * Adding Bicep template to monitoring addon
+  * Added custom DCR and DCE arm templates for Remote Write
+  * Adding monitoring reader role to Azure Monitor Workspace in ARM and Bicep templates
+  * Fix fluent-bit daemonset tailing path and mariner docs
+  * Liveness probe update for NON-MAC mode (windows)
```

Comment:
This release also adds windows daemonset support with MSI (only in deprecated chart mode)

Reply:
Thanks Kaveesh! Added this.